backend/src/models/reports/index.js

```diff
@@ -620,6 +620,108 @@ const getLessonsLearnedReport = (fiscalYear, projectId, portfolioIds) => {
   return results;
 };
 
+/**
+ * Gets the data for a Contract Summary Report
+ *
+ * @param   {number} contractId Contract id to limit report to.
+ * @returns {any}
+ */
+const getContractSummaryReport = (contractId) => {
+  return knex("data.contract as contract")
+    .select({
+      contract: "contract.*",
+      start_date: knex.raw(`TO_CHAR(contract.start_date :: DATE, '${dateFormat}')`),
+      end_date: knex.raw(`TO_CHAR(contract.end_date :: DATE, '${dateFormat}')`),
+      total_contract: knex.raw("contract.total_fee_amount + contract.total_expense_amount"),
+      supplier_name: "supplier_name",
+      internal_coding: "internal_coding.*",
+      portfolio: "portfolio.*",
+      portfolio_name: "portfolio.portfolio_name",
+    })
+    .leftJoin("data.supplier as supplier", { "supplier.id": "contract.supplier_id" })
+    .leftJoin("data.sid_internal_coding as internal_coding", {
+      "contract.id": "internal_coding.contract_id",
+    })
+    .leftJoin("data.portfolio as portfolio", { "portfolio.id": "internal_coding.portfolio_id" })
+    .where("contract.id", contractId);
+};
+
+/**
+ * Gets the contract invoices for a specific contract by id
+ *
+ * @param   {number} contractId Contract id to limit report to.
+ * @returns {any[]}
+ */
+const getContractInvoices = (contractId) => {
+  const results = knex("data.invoice as invoice")
+    .select({
+      fiscal: knex.min("fiscal_year"),
+      billing_period: knex.min("billing_period"),
+      invoice_date: knex.raw(`MIN(TO_CHAR(invoice_date:: DATE, '${dateFormat}'))`),
+      invoice_number: "invoice_number",
+      invoice_amount: knex.raw("SUM(unit_amount * rate)"),
```

Comment:
https://knexjs.org/guide/query-builder.html#sum i'm guessing because there is math within the sum that we can't use the knex.sum